charts/enforcer-k8s/templates/deployment.yaml

```
@@ -0,0 +1,65 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "enforcer-k8s.fullname" . }}
+  labels:
+    {{- include "enforcer-k8s.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "enforcer-k8s.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+    {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      labels:
+        {{- include "enforcer-k8s.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "enforcer-k8s.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default (printf "v%s" ($.Chart.AppVersion) )  }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          args:
+            - "--tls-secret={{ .Values.tlsSecretName }}"
```

the changes I'm making on the `config` branch expect this to be a namespaced name, so `${namespace}/${name}`.
```suggestion
            - "--tls-secret={{ .Release.Namespace }}/{{ .Values.tlsSecretName }}"
```